pyiron_contrib/atomistics/atomistics/job/trainingcontainer.py

```diff
@@ -93,6 +101,21 @@ def include_job(self, job, iteration_step=-1):
             forces = ff[iteration_step]
         else:
             forces = None
+
+        # Retrieve total charge.
+        tc = job.output.totalcharge
+        if tc is not None:
+            totalcharge = tc[iteration_step]
+        else:
+            totalcharge = None
+
+        # Retrieve atomic charges.
+        cc = job.output.charges
```

Comment:
I'm not sure right now, if you normal `AtomisticGenericJob.output` has these always defined, so you might need to make a `hasattr` check.